agent/server/index.js

```diff
@@ -11,19 +11,242 @@ import { sync as whichSync } from 'which';
 import bodyParser from 'body-parser';
 import { Server as WebSocketServer } from 'ws';
 
+const runChrome = () => {
```

Comment:
Can you isolate this 💩 to a separate module? I don't wanna look at it :trollface: